src/main.rs

```diff
@@ -62,6 +72,44 @@ fn error_exit(msg: &str) -> ! {
   std::process::exit(1);
 }
 
+fn usage() -> ! {
+  let usage = concat!(
+    "USAGE:\n",
+    "    cargo v <version>\n",
+    "    cargo v [options]\n",
+    "ARGUMENTS:\n",
+    "    version       Can be one of \"patch\", \"minor\", \"major\" or a string like \"v1.2.5\"\n",
+    "OPTIONS:\n",
+    "    -h, --help    Prints this message."
+  );
+
+  println!("{}", usage);
+  std::process::exit(0)
+}
+
+fn operation_of_string(arg: String) -> Operation {
```

Comment:
Maybe here we can just expect a `&str` instead of a owned `String`, since we are not mutating this argument

Reply:
I used `String` here because `args.next()` returns an `Option<String>`, so I kept the same type around. But it is better to make it expect a `&str`, will change this! 😄